cvpaperchallenge/limit-lab-homepage
# Enhancement/add web page basics

## Issue URL

close: #1 

## Change overview

- Docker-related files are added
- Configuration files necessary for frontend development are added, such as ESLint and Prettier settings.
- Basic web pages with fundamental layout and design are prepared.

## How to test

1. Boot-up a docker container

    ```bash
    docker compose up -d
    ```

1. Install dependencies

    ```bash
    npm install
    ```

1. Run a web application

    ```bash
    npm run dev
    ```

1. Run a code check

    ```bash
    # Check the code lint
    npm run lint
    ```

    ```bash
    # Check the code format
    npm run format
    ```

## Note for reviewers

N/A
